locust_tasks/locustfile.py

```diff
@@ -237,19 +377,19 @@ def enter_web_chat_query(self):
 class WebsiteUser(HttpUser):
     """
     This class controls the balance of the tasks which simulated users are performing.
-    TODO: Adjust to a more representative balance. (Currently set for development)
     """
     
     tasks = {
-        LaunchEQ: 100,
+        LaunchEQ: 0,
         LaunchEQInvalidUAC: 0,
         LaunchEQwithAddressCorrection: 0,
-        request_new_code: 0,
-        launch_web_chat: 0
+        RequestNewCodeSMS: 1,
+        RequestNewCodePost: 1,
+        LaunchWebChat: 0
     }
     
-    wait_time = between(2, 10)
-
+    #wait_time = between(2, 10)
+    wait_time = between(1, 1)
```

Comment:
A useful change for development but it needs to be the standardised '2, 10', so that test runs are directly comparable

Reply:
yes I'll change it back then